README.md

```diff
@@ -110,6 +110,10 @@ Please feel free to reach out to us in our [Discord](https://discord.gg/7Gc3DK33
 - Rootstock
 - Blast
 - Mode
+- Mantle
+- Darwinia
+- Darwinia Crab
```

Comment:
I think the standard name for this network is just Crab. [https://chainlist.org/?search=crab&testnets=true](https://chainlist.org/?search=crab&testnets=true)